### PR TITLE
Moving healthchecks to sashimi

### PR DESCRIPTION
**Pull request by johnsimons**
This PR moves the healthcheck endpoints to sashimi.

We also had to introduce testing Calamari commands directly without the server handler because certain projects reference really old libs that do not support netstandard!

### CODE REVIEW

File: source/Azure.Common/Sashimi.Azure.Common.csproj [hunk 1]
```diff
@@ -8,7 +8,8 @@
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <Nullable>enable</Nullable>
     <PackageProjectUrl>https://github.com/OctopusDeploy/Sashimi/</PackageProjectUrl>
-    <PackageLicenseUrl>https://github.com/OctopusDeploy/Sashimi/blob/master/LICENSE.txt</PackageLicenseUrl>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <LangVersion>default</LangVersion>
```
Comment (flin-8) [inline source/Azure.Common/Sashimi.Azure.Common.csproj:12]: I guess `PackageLicenseExpression` is the new thing?

Why do we have `LangVersion` for this file and not for `Sashimi.Aws.Common.csproj`

File: source/Sashimi.AzureServiceFabric/AzureServiceFabricAppHealthCheckActionHandler.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using Sashimi.Server.Contracts.ActionHandlers;
+using Sashimi.Server.Contracts.Calamari;
+
+namespace Sashimi.AzureServiceFabric
+{
+    class AzureServiceFabricAppHealthCheckActionHandler : IActionHandlerWithAccount
```
Comment (flin-8) [inline source/Sashimi.AzureServiceFabric/AzureServiceFabricAppHealthCheckActionHandler.cs:6]: Does this class also need to be registered in the autofac module? I see you've done that for WebApp and CloudService

File: source/Azure.Common/Sashimi.Azure.Common.csproj (hunk 1 above)
Comment (johnsimons) [reply source/Azure.Common/Sashimi.Azure.Common.csproj:12]: > I guess `PackageLicenseExpression` is the new thing?

Yes, it was complaining about it when we build with cake

> Why do we have LangVersion for this file and not for Sashimi.Aws.Common.csproj

Good point, I'm going to start a discussion in modularity slack to see where we want to go

File: source/Sashimi.AzureServiceFabric/AzureServiceFabricAppHealthCheckActionHandler.cs (hunk 2 above)
Comment (johnsimons) [reply source/Sashimi.AzureServiceFabric/AzureServiceFabricAppHealthCheckActionHandler.cs:6]: Fixed

**Pull request merged**